tmp/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM datadog/docker-dd-agent-build-rpm-x64:latest
+MAINTAINER Remi Hakim @remh
+
+# Update go to 1.10.3
+RUN curl -o /tmp/go1.10.3.linux-amd64.tar.gz https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz && \
```

Comment:
This is the Dockerfile I'll use to build the image that will get pushed to `datadog/docker-dd-agent-build-rpm-x64`

Reply:
I'll remove it before merging

Reply:
You don't have to remove it, add it to a more meaningful directory with a README or something, I think it's a legit way to keep track.